### PR TITLE
Added uin card template

### PR DESCRIPTION
**Pull request by dhanendra06**

### CODE REVIEW

File: uin_card_template.html [hunk 1]
```diff
@@ -0,0 +1 @@
...dress&nbsp;:&nbsp;</b></label> </td> <td><span>$!addressLine1_eng, $!addressLine2_eng, $!addressLine3_eng, $!region_eng, $!province_eng, $!city_eng, $!postalCode </span></td> </tr> <tr> <td><label class="name-head-color"><b>عنوان&nbsp;:&nbsp;</b></label> </td> <td><span>$!addressLine1_ara, $!addressLine2_ara, $!addressLine3_ara, $!region_ara, $!province_ara, $!city_ara, $!postalCode </span> </td> </tr> </table> </div> </div> </div> </td> </tr> </table> <div>&nbsp;</div> <table class="main-table" style="height: 300px"> <tr> <td> <div style="margin-left: 10px; margin-right: 10px; border: solid black 1px; height: 250px; width: 250px;"> <div class="col-md-6"> <div class="bar-code-padding"></div> <img src=$!QrCode style="width: 250px; height: 250px; margin-top: -20px"> </div> </div> </td> <td> <div class="name-head-color col-md-6"> <br> For any issues please contact us at<br> <br> Registration Proccessor,Hanging Gardens,Global Village Tech Park, Mysore Rd,RVCE,Bengaluru, Karnataka 560059<br> <br> لأية مشاكل يرجى الاتصال بنا على <br>التسجيل المعالج، الحدائق المعلقة ، القرية العالمية Tech Park, Mysore Rd,RVCE 560059 بنغالورو، كارناتاكا <br> <br> </div> </td> </tr> </table> </body> </html>
```
Comment (vishwa-vyom) [inline uin_card_template.html]: I remember Sasi mentioning about some encoding issue with plain template getting served from config service, is this config tested with multi lang support template getting serviced from config server ?

Comment (dhanendra06) [reply uin_card_template.html]: for testing, I have added @vishwa-vyom we need to check once after the merge.

File: digital-card-default.properties [hunk 2]
```diff
@@ -120,3 +120,5 @@ mosip.iam.adapter.renewal-before-expiry-interval=1440
 mosip.iam.adapter.self-token-renewal-enable=true
 mosip.auth.filter_disable=false
 auth.server.admin.allowed.audience=mosip-admin-client,mosip-resident-client
+
+mosip.digitalcard.service.template={RPR_UIN_CARD_TEMPLATE:uin_card_template.properties;vid-card-type:vid_card_template.properties}
```
Comment (loganathan-sekaran) [inline digital-card-default.properties]: Add detailed comment with example on the property value's format.

File: uin_card_template.properties [hunk 3]
```diff
@@ -0,0 +1 @@
...c3Bhbj4gPC90ZD4gPC90cj4gPC90YWJsZT4gPC9kaXY+IDwvZGl2PiA8L2Rpdj4gPC90ZD4gPC90cj4gPC90YWJsZT4gPGRpdj4mbmJzcDs8L2Rpdj4gPHRhYmxlIGNsYXNzPSJtYWluLXRhYmxlIiBzdHlsZT0iaGVpZ2h0OiAzMDBweCI+IDx0cj4gPHRkPiA8ZGl2IHN0eWxlPSJtYXJnaW4tbGVmdDogMTBweDsgbWFyZ2luLXJpZ2h0OiAxMHB4OyBib3JkZXI6IHNvbGlkIGJsYWNrIDFweDsgaGVpZ2h0OiAyNTBweDsgd2lkdGg6IDI1MHB4OyI+IDxkaXYgY2xhc3M9ImNvbC1tZC02Ij4gPGRpdiBjbGFzcz0iYmFyLWNvZGUtcGFkZGluZyI+PC9kaXY+IDxpbWcgc3JjPSQhUXJDb2RlIHN0eWxlPSJ3aWR0aDogMjUwcHg7IGhlaWdodDogMjUwcHg7IG1hcmdpbi10b3A6IC0yMHB4Ij4gPC9kaXY+IDwvZGl2PiA8L3RkPiA8dGQ+IDxkaXYgY2xhc3M9Im5hbWUtaGVhZC1jb2xvciBjb2wtbWQtNiI+IDxicj4gRm9yIGFueSBpc3N1ZXMgcGxlYXNlIGNvbnRhY3QgdXMgYXQ8YnI+IDxicj4gUmVnaXN0cmF0aW9uIFByb2NjZXNzb3IsSGFuZ2luZyBHYXJkZW5zLEdsb2JhbCBWaWxsYWdlIFRlY2ggUGFyaywgTXlzb3JlIFJkLFJWQ0UsQmVuZ2FsdXJ1LCBLYXJuYXRha2EgNTYwMDU5PGJyPiA8YnI+INmE2KPZitipINmF2LTYp9mD2YQg2YrYsdis2Ykg2KfZhNin2KrYtdin2YQg2KjZhtinINi52YTZiSA8YnI+2KfZhNiq2LPYrNmK2YQg2KfZhNmF2LnYp9mE2KzYjCDYp9mE2K3Yr9in2KbZgiDYp9mE2YXYudmE2YLYqSDYjCDYp9mE2YLYsdmK2Kkg2KfZhNi52KfZhNmF2YrYqSBUZWNoIFBhcmssIE15c29yZSBSZCxSVkNFIDU2MDA1OSDYqNmG2LrYp9mE2YjYsdmI2Iwg2YPYp9ix2YbYp9iq2KfZg9inIDxicj4gPGJyPiA8L2Rpdj4gPC90ZD4gPC90cj4gPC90YWJsZT4gPC9ib2R5PiA8L2h0bWw+
```
Comment (loganathan-sekaran) [inline uin_card_template.properties]: As discussed keep the properties in a single template properties file storing multiple templates as key value pairs in the same file.

File: registration-processor-default.properties [hunk 4]
```diff
@@ -473,8 +474,12 @@ mosip.biometric.sdk.providers.face.mosip-ref-impl-sdk-client.format.url.mock-1.1
 ## Printing
 mosip.registration.processor.credentialtype=euin
 mosip.registration.processor.encrypt=false
-mosip.registration.processor.issuer=mpartner-default-print 
 mosip.registration.processor.credential.request.service.id=mosip.credential.request.generator 
+mosip.registration.processor.digitalcard.credentialtype=PDFCard
+mosip.registration.processor.print.credentialtype=euin
+mosip.registration.processor.digitalcard.template=RPR_UIN_CARD_TEMPLATE
+mosip.registration.processor.print.template=RPR_UIN_CARD_TEMPLATE
+mosip.registration.processor.issuer=mpartner-default-print:${mosip.registration.processor.print.credentialtype}:${mosip.registration.processor.print.template};mpartner-default-digitalcard:${mosip.registration.processor.digitalcard.credentialtype}:${mosip.registration.processor.digitalcard.template}
```
Comment (loganathan-sekaran) [inline registration-processor-default.properties:484]: Add detailed comment with example on the property value's format.

**Review by loganathan-sekaran — CHANGES_REQUESTED** [review]

File: registration-processor-default.properties (hunk 4 above)
Comment (dhanendra06) [reply registration-processor-default.properties:484]: taken care

File: uin_card_template.properties (hunk 3 above)
Comment (dhanendra06) [reply uin_card_template.properties]: taken care

File: digital-card-default.properties (hunk 2 above)
Comment (dhanendra06) [reply digital-card-default.properties]: taken care

**Review by loganathan-sekaran — APPROVED** [review]

File: registration-processor-default.properties [hunk 5]
```diff
@@ -400,7 +401,7 @@ mosip.regproc.workflow.manager.message.expiry-time-limit=${mosip.regproc.common.
 #Audit request id
 mosip.registration.processor.audit.id=mosip.applicanttype.getApplicantType
 #Cryptomanager decrypt request id
-mosip.registration.processor.crypto.decrypt.id=mosip.cryptomanager.decrypt
+mosip.registration.processor.crypto.decrypt.id=mosip.cryptomanager.decryptuin_card_template.properties
```
Comment (vishwa-vyom) [inline registration-processor-default.properties]: why is this property change required ?

Comment (dhanendra06) [reply registration-processor-default.properties]: taken care

**Pull request merged**